docs/guide/apm-release-notes.asciidoc

```diff
@@ -2,6 +2,28 @@
 [[apm-release-notes]]
 == Release notes
 
+[float]
+==== APM version 6.5.0
```

Comment:
I might be wrong on this but isn't the correct name for the solution "Elastic APM" instead of just "APM"?

Reply:
Good question.  Leaving it as is for now as that's what we've had since this log started, I'll follow up on it.